### PR TITLE
Update AsyncGenerator to 0.14.0

### PR DESCRIPTION
**Pull request by maca88**
The new version contains an updated NUnit plugin that simplifies the integration with NUnit.

### CODE REVIEW

File: src/NHibernate.Test/Async/Ado/BatcherFixture.cs [hunk 1]
```diff
@@ -102,44 +102,7 @@ public async Task OneRoundTripUpdateAsync()
 			await (CleanupAsync());
 		}
 
-#if NETFX
-		[Test, Ignore("Not fixed yet.")]
-		[Description("SqlClient: The batcher should run all different INSERT queries in only one roundtrip.")]
-		public async Task SqlClientOneRoundTripForUpdateAndInsertAsync()
```
Comment (hazzik) [inline src/NHibernate.Test/Async/Ado/BatcherFixture.cs]: I didn't get it why this was removed.

Comment (maca88) [reply src/NHibernate.Test/Async/Ado/BatcherFixture.cs]: Until now the generator ignored only tests that had the `Ignore` attribute on the class, now I've applied the same rule also for methods.

**Comment by maca88** [comment]
I've changed to WIP until the issue in nhibernate/NHibernate-Caches/pull/65 is resolved. For some reason the issue does not happen for this project even when regenerating multiple times.

**Comment by maca88** [comment]
Removed WIP as the mentioned issue is not related to the generator.

**Comment by fredericDelaporte** [comment]
A conflict has appeared in the meantime, if you have not seen it.

**Comment by hazzik** [comment]
<details>
<summary>I'm getting a following error when try to execute async generator.</summary>

```text
     [exec] 10:28:09,975 INFO  AsyncCodeGenerator:0 - Transforming project 'NHibernate.Test' started
     [exec] 10:28:09,975 INFO  ProjectTransformer:0 - Generating documents started
     [exec] 10:28:12,951 FATAL CommandLine:0 - System.AggregateException: One or more errors occurred. ---> System.NullReferenceException: Object reference not set to an instance of an object.
     [exec]    at AsyncGenerator.Extensions.Internal.SyntaxNodeExtensions.ConstructNameSyntax(String name, Nullable`1 trailingTrivia, Boolean insideCref, Boolean onlyName)
     [exec]    at AsyncGenerator.Transformation.Internal.AsyncLockMethodTransformer.GetAsyncLockField(String fieldName, Boolean isStatic, IMethodOrAccessorTransformationResult result)
     [exec]    at AsyncGenerator.Transformation.Internal.AsyncLockMethodTransformer.Transform(IMethodOrAccessorTransformationResult result, ITypeTransformationMetadata typeMetadata, INamespaceTransformationMetadata namespaceMetadata)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.TransformMethod(MethodTransformationResult methodTransform, TypeDeclarationSyntax newTypeNode, TypeTransformationResult transformResult, INamespaceTransformationMetadata namespaceMetadata, SyntaxTrivia memberWhitespace, Boolean onlyMissingMembers)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.TransformMethodsAndProperties(TypeDeclarationSyntax newTypeNode, TypeTransformationResult transformResult, INamespaceTransformationMetadata namespaceMetadata, SyntaxTrivia memberWhitespace, Boolean onlyMissingMembers)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.TransformType(ITypeAnalyzationResult rootTypeResult, INamespaceTransformationMetadata namespaceMetadata, Boolean onlyMissingMembers)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.TransformType(ITypeAnalyzationResult rootTypeResult, INamespaceTransformationMetadata namespaceMetadata)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.TransformNamespace(INamespaceAnalyzationResult rootResult)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.TransformDocument(IDocumentAnalyzationResult documentResult)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.<>c__DisplayClass3_0.<Transform>g__TransfromDocument|0(IDocumentAnalyzationResult document)
     [exec]    at System.Threading.Tasks.Parallel.<>c__DisplayClass42_0`2.<PartitionerForEachWorker>b__1()
     [exec]    at System.Threading.Tasks.Task.InnerInvokeWithArg(Task childTask)
     [exec]    at System.Threading.Tasks.Task.<>c__DisplayClass176_0.<ExecuteSelfReplicating>b__0(Object )
     [exec]    --- End of inner exception stack trace ---
     [exec]    at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
     [exec]    at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
     [exec]    at System.Threading.Tasks.Parallel.PartitionerForEachWorker[TSource,TLocal](Partitioner`1 source, ParallelOptions parallelOptions, Action`1 simpleBody, Action`2 bodyWithState, Action`3 bodyWithStateAndIndex, Func`4 bodyWithStateAndLocal, Func`5 bodyWithEverything, Func`1 localInit, Action`1 localFinally)
     [exec]    at System.Threading.Tasks.Parallel.ForEachWorker[TSource,TLocal](IEnumerable`1 source, ParallelOptions parallelOptions, Action`1 body, Action`2 bodyWithState, Action`3 bodyWithStateAndIndex, Func`4 bodyWithStateAndLocal, Func`5 bodyWithEverything, Func`1 localInit, Action`1 localFinally)
     [exec]    at System.Threading.Tasks.Parallel.ForEach[TSource](IEnumerable`1 source, Action`1 body)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.Transform(IProjectAnalyzationResult analyzationResult)
     [exec]    at AsyncGenerator.AsyncCodeGenerator.<GenerateProject>d__2.MoveNext()
     [exec] --- End of stack trace from previous location where exception was thrown ---
     [exec]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
     [exec]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
     [exec]    at AsyncGenerator.AsyncCodeGenerator.<GenerateAsync>d__0.MoveNext()
     [exec] --- End of stack trace from previous location where exception was thrown ---
     [exec]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
     [exec]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
     [exec]    at AsyncGenerator.CommandLine.Program.Main(String[] args)
     [exec] ---> (Inner Exception #0) System.NullReferenceException: Object reference not set to an instance of an object.
     [exec]    at AsyncGenerator.Extensions.Internal.SyntaxNodeExtensions.ConstructNameSyntax(String name, Nullable`1 trailingTrivia, Boolean insideCref, Boolean onlyName)
     [exec]    at AsyncGenerator.Transformation.Internal.AsyncLockMethodTransformer.GetAsyncLockField(String fieldName, Boolean isStatic, IMethodOrAccessorTransformationResult result)
     [exec]    at AsyncGenerator.Transformation.Internal.AsyncLockMethodTransformer.Transform(IMethodOrAccessorTransformationResult result, ITypeTransformationMetadata typeMetadata, INamespaceTransformationMetadata namespaceMetadata)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.TransformMethod(MethodTransformationResult methodTransform, TypeDeclarationSyntax newTypeNode, TypeTransformationResult transformResult, INamespaceTransformationMetadata namespaceMetadata, SyntaxTrivia memberWhitespace, Boolean onlyMissingMembers)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.TransformMethodsAndProperties(TypeDeclarationSyntax newTypeNode, TypeTransformationResult transformResult, INamespaceTransformationMetadata namespaceMetadata, SyntaxTrivia memberWhitespace, Boolean onlyMissingMembers)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.TransformType(ITypeAnalyzationResult rootTypeResult, INamespaceTransformationMetadata namespaceMetadata, Boolean onlyMissingMembers)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.TransformType(ITypeAnalyzationResult rootTypeResult, INamespaceTransformationMetadata namespaceMetadata)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.TransformNamespace(INamespaceAnalyzationResult rootResult)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.TransformDocument(IDocumentAnalyzationResult documentResult)
     [exec]    at AsyncGenerator.Transformation.Internal.ProjectTransformer.<>c__DisplayClass3_0.<Transform>g__TransfromDocument|0(IDocumentAnalyzationResult document)
     [exec]    at System.Threading.Tasks.Parallel.<>c__DisplayClass42_0`2.<PartitionerForEachWorker>b__1()
     [exec]    at System.Threading.Tasks.Task.InnerInvokeWithArg(Task childTask)
     [exec]    at System.Threading.Tasks.Task.<>c__DisplayClass176_0.<ExecuteSelfReplicating>b__0(Object )<---
     [exec]
```
</details>

**Pull request merged**